code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -5,7 +5,7 @@ var/list/forbidden_varedit_object_types = list(
 										/datum/admin_rank					//editing my own rank? it's more likely than you think
 									)
 
-var/list/VVlocked = list("vars", "client", "virus", "viruses", "cuffed", "last_eaten", "unlock_content", "step_x", "step_y", "force_ending")
+var/list/VVlocked = list("vars", "var_edited", "client", "virus", "viruses", "cuffed", "last_eaten", "unlock_content", "step_x", "step_y", "force_ending")

```

Comment:
This is technically redundant, as any attempts to set var_edited to 0 again would set it to 1 as it is, in itself, a var edit.

It's more just to keep the var list a little tidier.